transmogrifier/sources/xml/ead.py

```diff
@@ -499,6 +484,30 @@ def _parse_date_elements(
                 dates.append(date_instance)
         return dates
 
+    @classmethod
+    def _parse_date_range(cls, date_string: str, source_record_id: str) -> timdex.Date:
+        date_instance = timdex.Date()
+        gte_date, lte_date = date_string.split("/")
```

Comment:
Slightly different approach from `Datacite` to instantiate a separate date instance in the method rather than pass the existing one from `get_dates` but as Graham said we'll certainly be evaluated and hopefully centralizing a lot of the date parsing functionality from out of the source transform. We can decide on the preferred approach then!

Reply:
Sounds good!